SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -70,169 +70,142 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	uint32 EntityPoolInitialReservationCount;
 
 	/**
-	 * Specifies when the SpatialOS Runtime should reserve a new batch of entity IDs: the value is the number of un-used entity
-	 * IDs left in the entity pool which triggers the SpatialOS Runtime to reserve new entity IDs
+	 * The number of remaining entity IDs in the pool that triggers a new batch.
 	 */
 	UPROPERTY(EditAnywhere, config, Category = "Entity Pool", meta = (DisplayName = "Pool Refresh Threshold"))
 	uint32 EntityPoolRefreshThreshold;
 
 	/**
-	 * Specifies the number of new entity IDs the SpatialOS Runtime reserves when `Pool refresh threshold` triggers a new batch.
+	 * The number of entity IDs in a new batch.
 	 */
 	UPROPERTY(EditAnywhere, config, Category = "Entity Pool", meta = (DisplayName = "Refresh Count"))
 	uint32 EntityPoolRefreshCount;
 
 	/**
-	 * Specifies the amount of time, in seconds, between heartbeat events sent from a game client to notify the server-worker instances
-	 * that it's connected.
+	 * From game clients to server-workers.
 	 */
-	UPROPERTY(EditAnywhere, config, Category = "Heartbeat", meta = (DisplayName = "Heartbeat Interval (seconds)"))
+	UPROPERTY(EditAnywhere, config, Category = "Heartbeat", meta = (DisplayName = "Heartbeat Interval (Seconds)"))
 	float HeartbeatIntervalSeconds;
 
-	/**
-	 * Specifies the maximum amount of time, in seconds, that the server-worker instances wait for a game client to send heartbeat events.
-	 * (If the timeout expires, the game client has disconnected.)
-	 */
-	UPROPERTY(EditAnywhere, config, Category = "Heartbeat", meta = (DisplayName = "Heartbeat Timeout (seconds)"))
+
+	UPROPERTY(EditAnywhere, config, Category = "Heartbeat", meta = (DisplayName = "Heartbeat Timeout (Seconds)"))
 	float HeartbeatTimeoutSeconds;
 
 	/**
-	 * Same as HeartbeatTimeoutSeconds, but used if WITH_EDITOR is defined.
+	 * Use when you are running the game server from the editor, or with the editor executable.
 	 */
-	UPROPERTY(EditAnywhere, config, Category = "Heartbeat", meta = (DisplayName = "Heartbeat Timeout With Editor (seconds)"))
+	UPROPERTY(EditAnywhere, config, Category = "Heartbeat", meta = (DisplayName = "Heartbeat Timeout With Editor (Seconds)"))
 	float HeartbeatTimeoutWithEditorSeconds;
 
 	/**
-	 * Specifies the maximum number of Actors replicated per tick. Not respected when using the Replication Graph.
-	 * Default: `0` per tick  (no limit)
-	 * (If you set the value to ` 0`, the SpatialOS Runtime replicates every Actor per tick; this forms a large SpatialOS  world, affecting
-	 * the performance of both game clients and server-worker instances.) You can use the `stat Spatial` flag when you run project builds to
-	 * find the number of calls to `ReplicateActor`, and then use this number for reference.
+	 * Warning: default value 0 means that every Actor replicates. Ignored when using Replication Graph.
```

Comment:
* Warning: default value 0 means that every Actor replicates - this can affect the performance of your game.  Tip: Use the number of calls to `ReplicateActor` for this value (to find the number of calls, use the`stat Spatial` flag when you build your project from the command line). Note: This setting is Ignored when using Replication Graph. 

//Suggest adding why replication of every Actor is bad - even though I know it should be obvious -  and the tip on what is a good value is helpful?!?